Assets/_BForBoss/_Core/Scripts/Boss/DerekMissileBehaviour.cs

```diff
@@ -151,5 +153,25 @@ private void OnEnable()
             _elapsedAutoPilotTimeToLive = 0;
             _state = State.TowardsApex;
         }
+
+        private void Start()
+        {
+            //NOTE - *Physics.IgnoreCollision*
+            //Note that this code should be run in the Start() method or later, since the colliders need to be initialized before you can use them. 
+            var missileCollider = GetComponent<Collider>();
+            var deathAreas = FindObjectsOfType<DeathAreaBehaviour>()
```

Comment:
Do we want to make sure we handle the Inactive `DeathAreaBehaviors` as well, since not all of them will active at the same time

I think its like `FindObjectsOfType<DeathAreaBehavior>(includeInctive: true)`

Reply:
oh yeah you're right.